src/mumble/AudioWizard.cpp

```diff
@@ -355,14 +355,21 @@ int AudioWizard::nextId() const {
 }
 
 void AudioWizard::playChord() {
+	if (!m_chordEnabled) {
+		return;
+	}
+
 	AudioOutputPtr ao = Global::get().ao;
-	if (!ao || aosSource || bInit)
+
+	if (!ao || m_chord != AudioOutputToken::None() || bInit) {
```

Comment:
```suggestion
	if (!ao || m_chord || bInit) {
```
Use new conversion to bool